### PR TITLE
MDS-101 TW Drawer component

### PR DESCRIPTION
**Pull request by oykos-development-hub**
## Checklist

- [ ] You attached screenshots or added description about changes
- [ ] You use [conventional commits naming](https://www.conventionalcommits.org/en/v1.0.0/), e.g. `fix: your changes description [TicketNumber]`, `feat: your feature name [TicketNumber]`
- [ ] You have updated the documentation accordingly.

### CODE REVIEW

File: workspaces/core/src/drawer/Drawer.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  content,
+  primaryButton,
+  secondaryButton,
+  tertiaryButton,
+  handleToggle
+}) => {
+  return (<div>
+    {isOpen && (<div
+      className='flex fixed top-0 left-0 h-[100vh] w-full justify-end items-stretch z-10 cursor-pointer '
+      style={{ backgroundColor: 'rgba(0, 0, 0, 0.56)'}}
```
Comment (yarema184) [inline workspaces/core/src/drawer/Drawer.tsx:20]: Please create a class for this color in moon-preset and call it `backdrop`

File: next-docs/public/examples/drawerTW/Default.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { Drawer, Button } from '@heathmont/moon-core-tw';
+import React, {useState} from 'react';
+
+const Example = () => {
```
Comment (yarema184) [inline next-docs/public/examples/drawerTW/Default.tsx:4]: Please add an example without buttons

File: next-docs/pages/core/drawer.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            default: '-',
+            description: 'Text for footer button',
+          },
+          {
+            name: 'buttonThreeLabel',
+            type: 'string',
+            required: false,
+            default: '-',
+            description: 'Text for footer button',
+          },
```
Comment (yarema184) [inline next-docs/pages/core/drawer.tsx:63]: Please add correct props  and descriptions to the Table

File: workspaces/core/src/drawer/Drawer.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        if (handleToggle) handleToggle(false);
+      }}
+    />)}
+    <div
+      // TW not working for these classes, don't know why?
+      style={{
+        width: `${width}vw`,
+        height: `${height}vh`,
+      }}
+      className={`w-[${width}vw] h-[${height}vh] flex flex-col items-stretch justify-between bg-white z-[99] fixed ease-in-out duration-300 top-0 left-[100%] shadow-xl z-20 ${
```
Comment (yarema184) [inline workspaces/core/src/drawer/Drawer.tsx:34]: 'z-[99]' applies the same CSS properties as 'z-20'

**Review by yarema184 — CHANGES_REQUESTED** [review]

File: next-docs/tailwind.config.js [hunk 5]
```diff
@@ -28,6 +28,7 @@ module.exports = {
         developer: 'rgba(149, 241, 213, 0.9)',
         transparent: 'transparent',
         current: 'currentColor',
+        backdrop: 'rgba(0, 0, 0, 0.56)'
```
Comment (yarema184) [inline next-docs/tailwind.config.js:31]: What is the reason to replace backdrop color in next-docs/tailwind.config.js as it was added in moon-preset ?

Comment (oykos-development-hub) [reply next-docs/tailwind.config.js]: removed

**Review by yarema184 — APPROVED** [review]

**Pull request closed without merging**